source/brailleDisplayDrivers/seikantk.py

```diff
@@ -290,6 +286,18 @@ def __init__(self, index):
 		self.routingIndex = index
 
 
+def _getKeyNames(keys: int) -> Set[int]:
+	return {_keyNames[1 << i] for i in range(16) if (1 << i) & keys}
```

Comment:
I think this (and the other similar methods) deserve an explanation. This seems to be converting a bit set flag `keys` into a set of key names. Given it needs to iterate over the full 16 values, you might as well invert this to avoid the hard to read bit flipping.

```suggestion
	return {keyName for bitFlag, keyName in _keyNames.items() if bitFlag & keys}
```

I think this is safer too, it is only testing for bits that we have known values for.